google/cloud/bigquery/v2/minimal/internal/job_logging.cc

```diff
@@ -32,12 +33,15 @@ template <typename Functor, typename Request,
               Functor, rest_internal::RestContext&, Request const&>>
 Result JobLogWrapper(Functor&& functor, rest_internal::RestContext& context,
                      Request const& request, char const* where,
+                     absl::string_view request_name,
+                     absl::string_view response_name,
                      TracingOptions const& options) {
   auto formatter = [](std::string* out, auto const& header) {
     absl::StrAppend(out, "name=", header.first, ", value={",
                     absl::StrJoin(header.second, "&"), "}");
   };
-  GCP_LOG(DEBUG) << where << "() " << request << ", Context={"
+  GCP_LOG(DEBUG) << where << "() << "
+                 << request.DebugString(request_name, options) << ", Context={"
                  << internal::DebugString(
```

Comment:
Typically we want to apply the truncation to string values rather than formatted elements, so you'd get output like ...
```
  Context={name="name", value={"foo&ba...<truncated>..."}}
```
instead of ...
```
  Context={name="...<truncated>...
```